packages/entity-detail/src/components/DetailForm/DetailForm.js

```diff
@@ -91,9 +93,9 @@ export class DetailForm extends React.Component {
           />
           {!this.isReadOnlyForm()
           && <div>
-            {!props.valid && props.anyTouched && <ErrorBox formErrors={props.formErrors} showErrors={this.showErrors}/>}
+            {!props.valid && props.isDirty && <ErrorBox formErrors={props.formErrors} showErrors={this.showErrors}/>}
```

Comment:
Not quite perfect. 
Reproduce problem:
In the mocked form touch, but not edit, the "Callname" Field. Save button is not disabled but error on field is shown.

![image](https://user-images.githubusercontent.com/7300481/56190197-0036e000-602a-11e9-876d-0b17b54dc9e3.png)